tests/test_blotter.py

```diff
@@ -112,6 +114,25 @@ def test_blotter_order_types(self, style_obj, expected_lmt, expected_stp):
         self.assertEqual(result.limit, expected_lmt)
         self.assertEqual(result.stop, expected_stp)
 
+    def test_blotter_eod_cancellation(self):
```

Comment:
do you also need a test for the blotter not canceling orders if its policy is `NeverCancel`?